### ISSUE TITLE
Conda and Mamba Installation Problem. Urgent!

### ISSUE THREAD

**Issue by drorhunvural** [Discngine/fpocket#123]
**Bug**

There hasn't been any issue with using _!conda install fpocket_ or _!mamba install fpocket_ for a long time. Currently, both of them are giving errors, and the installation process cannot be completed. Fpocket can no longer be used in this way on Google Colab. The problem is caused by the latest update of Google Colab.  You can track the Cudatoolkit issue [here](
https://github.com/googlecolab/colabtools/issues/4214).

Updating Fpocket to be compatible with the latest version of Google Colab will make it usable again. I believe that you will implement the necessary modifications for this issue.

**Screenshots**

![image](https://github.com/Discngine/fpocket/assets/26056864/1c7f2605-de1c-42e6-9718-d8ff8e7cf4e1)

![image](https://github.com/Discngine/fpocket/assets/26056864/499a8046-4da7-4621-a121-9baa069b46bf)

**Comment by pschmidtke** [comment]
Why on earth is cuda necessary here?
@simonbray do you happen to have an idea by any chance :)

**Comment by simonbray** [comment]
> Why on earth is cuda necessary here? @simonbray do you happen to have an idea by any chance :)

My best guess is that that cudatoolkit and fpocket have a common dependency somewhere but require different versions of it. I don't think this is our job to fix though.

I don't know anything about Google Colab but in general I would create a new conda environment and installing fpocket there.

**Comment by pschmidtke** [comment]
indeed looks like google colab dep issue, not fpocket. I'll try a conda install fpocket on an isolated env to double check

**Comment by pschmidtke** [comment]
the latest one still needs to be pushed, but fpocket on conda is working if run on a local conda. So as @simonbray suggested, looks like an issue on colab or other, not fpocket related. 
Will close the issue here

**Comment by drorhunvural** [comment]
> Why on earth is cuda necessary here?

As you stated and I stated in the question, this issue is not with Fpocket, but rather due to Google Colab's recent update of CUDA. The reason why I opened this problem is that Fpocket cannot be used in Google Colab right now. Not everyone has easy access to GPUs in their projects, which is why millions of people use Google Colab. Most libraries take action themselves to be compatible with Google Colab.
